src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -578,21 +616,21 @@ private void CheckStake(Block block, ChainedBlock chainTip)
             {
                 this.rpcGetStakingInfoModel.Staking = false;
 
-                this.logger.LogTrace("Total balance of available UTXOs is {0}, which is lower than reserve balance {1}.", balance, this.reserveBalance);
+                this.logger.LogTrace("Total balance of available UTXOs is {0}, which is equal or lower than reserve balance {1}.", balance, this.reserveBalance);
```

Comment:
`lower or equal` (think `<=` it is first `<` then `=`) and probably best is `less than or equal to`